Resources/Private/Templates/Feed.html

```
@@ -1,38 +1,83 @@
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    href="https://twitter.com/{tweet.username}"
+                                    title="{tweet.name}">
+                                    @{tweet.username}
+                                </a>
+                                <span class="text-muted small ms-auto">
+                                    {tweet.date->f:format.date(format: 'd.m.Y')}
+                                </span>
+                            </div>
+
+                            <p class="mb-2 twitter-feed__text">{tweet.textAsHtml->f:format.raw()}</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)